### PR TITLE
Smash and graaagh

### PR DESCRIPTION
**Pull request by Tokoeka**
Add a preference and tracking for the Zombie Slayer skill Smash & Graaagh

### CODE REVIEW

**Comment by Rinn** [comment]
imo i would treat it as limited because there's no benefit to using this skill after the pickpockets stop working

File: src/net/sourceforge/kolmafia/request/FightRequest.java [hunk 1]
```diff
@@ -9210,6 +9210,13 @@ private static void payActionCost(final String responseText) {
         }
         break;
 
+      case SkillPool.SMASH_GRAAAGH:
+        if (responseText.contains("incidentally tearing free an item")) {
+          // No casting limit, can pickpocket items up to 30 times a day
+          Preferences.increment("_zombieSmashPocketsUsed");
```
Comment (gausie) [inline src/net/sourceforge/kolmafia/request/FightRequest.java:9216]: If you've set a daily limit you want to set skillSuccess = true here instead

File: src/net/sourceforge/kolmafia/request/FightRequest.java [hunk 2]
```diff
@@ -9210,6 +9210,14 @@ private static void payActionCost(final String responseText) {
         }
         break;
 
+      case SkillPool.SMASH_GRAAAGH:
+        if (responseText.contains("incidentally tearing free an item")) {
+          // No casting limit, can pickpocket items up to 30 times a day
+		  skillSuccess = true;
```
Comment (Rinn) [inline src/net/sourceforge/kolmafia/request/FightRequest.java:9216]: got some weird spacing here

**Review by Rinn — APPROVED** [review]

**Comment by gausie** [comment]
Thanks for this! Merging

**Pull request merged**